### PR TITLE
Scheduler edits

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/api-testing/schedule-test.md [hunk 1]
```diff
@@ -18,19 +18,22 @@ The **Schedule** tool, accessible from within each test, allows you to schedule
 ## Creating a New Test Schedule
 
 :::caution Required Step
-Do not proceed until you've [published the working copy of your test](/api-testing/quickstart/#publish-your-test).
+Schedule is available only for [published tests](/api-testing/quickstart/#publish-your-test).
```
Comment (kimsauce) [inline docs/api-testing/schedule-test.md:21]:
```suggestion
The **Schedule** functionality is available only for [published tests](/api-testing/quickstart/#publish-your-test).
```

File: docs/api-testing/schedule-test.md [hunk 2]
```diff
@@ -18,19 +18,22 @@ The **Schedule** tool, accessible from within each test, allows you to schedule
...
 :::
 
-1. Click the **Schedule** tab to access the Scheduler.<br/><img src={useBaseUrl('img/api-fortress/2021/02/scheduleTab.png')} alt="Schedule Tab"/>
-2. Click **Create Schedule**.<br/><img src={useBaseUrl('img/api-fortress/2021/02/createSchedule.png')} alt="Create Schedule" width="450"/>
+1. From within the **Composer**, click the **Schedule** tab to access the Scheduler.<br/><img src={useBaseUrl('img/api-fortress/2021/02/scheduleTab.png')} alt="Schedule Tab"/>
```
Comment (kimsauce) [inline docs/api-testing/schedule-test.md:24]: @LoryApiFortress , I think we may have discussed before -- we should be using **Compose** and **Schedule** (not  **Composer** and **Scheduler**) to match the UI elements. 

You could also speak with the Growth Team to propose changing the UI elements to **Composer** and **Scheduler**.

**Review by kimsauce — DISMISSED** [review]
Some recommendations

**Review by lauramthomas — CHANGES_REQUESTED** [review]
@LoryApiFortress I reviewed your changes and made some of my own.

**Review by lauramthomas — APPROVED** [review]

**Pull request merged**